packages/blaze/template.js

```diff
@@ -39,6 +45,9 @@ Blaze.Template = function (viewName, renderFunction) {
     rendered: [],
     destroyed: []
   };
+
+  options = options || {};
+  this._formalArgs = options.formalArgs || [];
```

Comment:
How do you differ between one not specifying the new arguments to the template. And one wanting to enable new feature, but with an empty list? So that one can access passed arguments inside `.args`, but otherwise template works as it was before? Or will this be simply a default? So that you can always call a template with `{{@> template a='foo'`}}`, even if it did not specify any arguments and those arguments are available in the `.args`?

Then there is no difference between `<template name="foo" args="">` and `<template name="foo">`? In fact, I like that. If this would be like that, then please keep it like this. Then I do not need API to set template arguments at runtime. :-) My main concern was that I want to enable new nature of templates without having to modify the template.

But here you are introducing `@>` and in the proposal the idea was that caller would not have to think about this? But then how do you enable empty args list on a template?